data/Tolsma_2007/metadata.yml

```diff
@@ -0,0 +1,468 @@
+  replicates: .na
+  methods: Data taken from McCarthy et al. (2000), with exception of Erigeron nitidus
+    and Stackhousia monogyna (mode assumed same as closely related species). 
+- var_in: root_morphology
+  unit_in: .na
+  trait_name: .na
+  entity_type: population
+  value_type: mode
+  basis_of_value: expert_score
+  replicates: 3
+  methods: During January 2001 (mid-summer), samples of root tissue were taken from
+    37 perennial monocotyledonous and dicotyledonous grassland species common on the
+    plateau (Table 1), including small shrubs scattered within the grassland. Most
+    species were in flower, allowing accurate identification, and were assumed to
+    be at a similar phenological stage. All samples were taken from an area regenerating
+    after the Caledonia Fire 3 years previously, and so reseeding species were 3 years
+    old or younger, while resprouting species had regenerated from basal or root buds.
+    No plants were more than 30 cm high. Collection of root material occurred between
+    1200 and 1400 hours on days of similar weather condition (sunny, and ~20°C). Three
+    plants of each species were gently levered out of the ground sufficiently to characterise
+    their root morphology and average maximum root diameter. Plants were allocated
+    to four categories of root morphology, based on increasing fleshiness -- (a) strong
+    woody, (b) firm, (c) thin fleshy (soft, ≤3 mm average diameter), and (d) thick
+    fleshy (soft, ≥4 mm average diameter).
+- var_in: root_woodiness
+  unit_in: .na
+  trait_name: woodiness_detailed
+  entity_type: population
+  value_type: mode
+  basis_of_value: expert_score
+  replicates: 3
+  methods: During January 2001 (mid-summer), samples of root tissue were taken from
+    37 perennial monocotyledonous and dicotyledonous grassland species common on the
+    plateau (Table 1), including small shrubs scattered within the grassland. Most
+    species were in flower, allowing accurate identification, and were assumed to
+    be at a similar phenological stage. All samples were taken from an area regenerating
+    after the Caledonia Fire 3 years previously, and so reseeding species were 3 years
+    old or younger, while resprouting species had regenerated from basal or root buds.
+    No plants were more than 30 cm high. Collection of root material occurred between
+    1200 and 1400 hours on days of similar weather condition (sunny, and ~20°C). Three
+    plants of each species were gently levered out of the ground sufficiently to characterise
+    their root morphology and average maximum root diameter. Plants were allocated
+    to four categories of root morphology, based on increasing fleshiness -- (a) strong
+    woody, (b) firm, (c) thin fleshy (soft, ≤3 mm average diameter), and (d) thick
+    fleshy (soft, ≥4 mm average diameter).
+- var_in: mean_maximum_root_diameter
+  unit_in: mm
+  trait_name: root_diameter
+  entity_type: population
+  value_type: mean
```

Comment:
Trying to decide if this shouldn't instead be max? To make absolutely clear that it is a max, not a mean value?